src/UI/MainMenuOptions.cpp

```diff
@@ -70,27 +70,37 @@ void MainMenuOptions::init()
     cmbResolution.maxDisplayItems(1);
 
     cmbResolution.clearSelection();
+    { //Default resolution
+        std::ostringstream ss;
+        ss << constants::MINIMUM_WINDOW_WIDTH << 'x' << constants::MINIMUM_WINDOW_HEIGHT << 'x' << 60;
+        cmbResolution.text(ss.str());
```

Comment:
Consider using [`std::to_string`](https://en.cppreference.com/w/cpp/string/basic_string/to_string) rather than `std::ostringstream`.

Reply:
Is the a difference?

Reply:
The `std::to_string` calls can be inlined, allowing this to be done in 1 line of code. The `ostringstream` needs to be declared first, then used, then have a `str()` extracted from it.